cryptlex/cryptlex-enterprise/values.yaml

```diff
@@ -88,23 +88,75 @@ services:
       repository: cryptlex/admin-portal
       tag: 5.2.3
       pullPolicy: IfNotPresent
+    # Pod security context
+    podSecurityContext:
+      runAsUser: 1000
+      runAsGroup: 1000
+      runAsNonRoot: true
+    # Container security context
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+          - ALL
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true  
   customerPortal:
     enabled: false
     image:
       repository: cryptlex/customer-portal
```

Comment:
@mumin-khan are the web portal conatiner images using non-root, how did we get 1000 value